solr/solr-ref-guide/src/authentication-and-authorization-plugins.adoc

```diff
@@ -1,4 +1,4 @@
-= Authentication and Authorization Plugins
+= Configuring Authentication, Authorization and Auditlogging plugins
```

Comment:
I have not renamed the file itself since it WILL break a ton of URL links and I'm not sure how to efficiently do it in asciidoc. I first named the page "Configuring security plugins in security.json" but then changed the name to make it more searchable and so that it will still pop up for "auth" in the title-search in online refGuide.

However, if we have figured out how to do redirects from old pages to new ones, we could rename the adoc file as well?

Reply:
I prefer keeping the "auth" in the title, for the same reasons. I don't think we've worked out how to do redirects well - we don't really even have a functional 404 page. It's fine to keep the filename the same since it includes most of the reworked title, in my opinion, so I'm not worried about keeping it the way it is already. (We should figure out redirects, but it doesn't have to be done here.)

Do you mean to use "Auditlogging" in the title though? It's a bit inconsistent in these edits - sometimes it's "auditlogging" and sometimes "audit logging". I feel like it should be "audit logging" wherever possible. An exception would be when referring to a class, and then I'll guess it would be something like "AuditLoggingPlugin".

I think we could drop "Plugins" from the page title to be honest. The page is about configuring authentication, authorization and audit logging - the fact that's done with plugins doesn't need to be in the page title IMO. If you decide to keep "plugins" in the title, though, it should be capitalized so we maintain our standard of headline case for page and section titles.